script/release-docker.sh

```
@@ -0,0 +1,4 @@
+docker build --tag ghcr.io/0235-jp/karkuri-agent-dev:latest \
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)